lib/core/src/lib/datatable/components/datatable/datatable.component.spec.ts

```diff
@@ -1844,6 +1844,15 @@ describe('Column Resizing', () => {
         expect(adapter.setColumns).toHaveBeenCalledWith(columns);
     }));
 
+    it('should prevent default behaviour on row click event', () => {
+        const clickEvent = new MouseEvent('click');
+        spyOn(clickEvent, 'preventDefault');
```

Comment:
Based on your changes in component I think you should check if stopPropagation is called, not preventDefault, because you use stopPropagation in ts file.